asf_heat_pump_affordability/pipeline/generate_cost_percentiles.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+import numpy as np
+
+
+def generate_dict_cost_percentiles_by_archetype(
+    cost_series: pd.Series,
+    archetypes_masks: dict,
+    percentiles: list or np.array,
```

Comment:
This could be more [generic](https://mypy.readthedocs.io/en/stable/builtin_types.html#generic-types), e.g.

`from typing import Iterable`

`percentiles: Iterable[float]`

Reply:
Oh yeah thanks!